src/metabase/db/custom_migrations.clj

```diff
@@ -1683,6 +1683,17 @@
       (when (not= pmappings orig-pmappings)
         pmappings))))
 
+(defn- card->last-stage
+  [record]
```

Comment:
:x: Parameter mapping disappears in the UI after running the migration.
Reason unclear.
It may have something to do with object keys not matching `target.dimension` - we should regenerate those.

![image](https://github.com/user-attachments/assets/e011d46b-9668-47d1-882f-6d82fe00805f)

Reply:
Added conversion for the `id` fields and keys.

Reply:
This is fixed now